.github/workflows/superset-frontend.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
         uses: actions/checkout@v4
         with:
+          fetch-depth: 0 # Fetch all history
```

Comment:
hey, curious why this was/is neeeded?

Reply:
It was indeed to get around this:

![image](https://github.com/apache/superset/assets/812905/bfd02639-bd6e-4763-bd90-2df752a8ff6a)